<launch>

    <arg name="environment" default="sea" />
    <arg name="feedback_method" default="attitude_with_depth" />

    <include file="$(find imu_complementary_filter)/launch/bluerov_complimentary_filter.launch" />

    <node name="tf_broadcaster" pkg="bluerov_control_ros" type="bluerov_tf_broadcaster">
        <param name="environment" value="$(arg environment)" />
        <param name="feedback_method" value="$(arg feedback_method)" />

        <remap from="attitude_topic" to="/imu_tools/attitude/filtered" />
        <remap from="pressure_topic" to="/mavros/imu/static_pressure" />
    </node>
    
</launch>